### ISSUE TITLE
Publish the plugin to Maven Central

### ISSUE THREAD

**Issue by boris-petrov** [jruby-gradle/jruby-gradle-plugin#420]
[Bintray+JCenter is being terminated](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) so the path forward would be to use Maven Central (at least for now - we're still waiting on official information from the Gradle team). There is a nice summary on what must be done [here](https://mvysny.github.io/jcenter-is-dead/).

**Comment by ysb33r** [comment]
No, these plugins should be published directly to the Gradle Plugin Portal

**Comment by boris-petrov** [comment]
Or that, yes. Not sure if there are any limitations to that. If you manage to do it and want to write the steps down, please share it here. Thanks!

**Comment by ysb33r** [comment]
I know how to do that, but there are some process decisions to be made.

. How is going to own the portal account and keys. Although I can do that I fee that the JRuby team is better suited to this. (@headius ?)
. I am waiting to see what the Gradle team is going to work out about migrating older artifacts that were using the Bintray sync mechanism.

**Comment by headius** [comment]
JRuby team would be the right group to own these credentials but we need some help managing this plugin.

**Comment by ysb33r** [comment]
@rtyler First thing is to recover the credentials to publish to Gradle portal, once you have that then we can add them as protected variables in GH and use a GH Actions to publish

**Comment by rtyler** [comment]
`PUBLISH_KEY` and `PUBLISH_SECRET` added to GitHub Actions. Would that be enough to help you take this from here @ysb33r ?

**Comment by ysb33r** [comment]
@rtyler I made a bunch of progress, but it is curretnly failing with

```
Execution failed for task ':jruby-gradle-core-plugin:publishPlugins'.
> Unauthorized!
    Login from command line using:
      gradle login
    or login and download user API keys on:
      https://plugins.gradle.org/user/login
```
(https://github.com/jruby-gradle/jruby-gradle-plugin/runs/2487940190?check_suite_focus=true)

I'm not sure yet whether the credentials are incorrect or whether they are not passed in correctly in the `publish` job.

Maybe someone with more GHA-fu can confirm whether the repository secrets are being used correctly: https://github.com/jruby-gradle/jruby-gradle-plugin/blob/c673c36e20887c56df4a0bc10dee0aac873a7714/.github/workflows/build.yml#L140-L178

**Comment by UweKubosch** [comment]
I am looking at getting a release out.  What is the status here?

**Comment by boris-petrov** [comment]
Actually, I'm not sure. I believe it is resolved - that is, the latest version is using the Gradle Plugin Portal and Maven Central for repositories - I don't see JCenter anywhere. So I think this can be closed!
